docs/model_server_rest_api_kfs.md

```diff
@@ -0,0 +1,266 @@
+# KServe compatible RESTful API {#ovms_docs_rest_api_kfs}
+
+## Introduction
+In addition with [gRPC APIs](./model_server_grpc_api_kfs.md) OpenVINO&trade; model server also supports RESTful APIs which follows the documentation from [KServe REST API](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/required_api.md#httprest). REST API is recommended when the primary goal is in reducing the number of client-side python dependencies and simpler application code.
+
+This document covers the following API:
+* <a href="#kfs-server-live">Server Live API </a>
+* <a href="#kfs-server-ready">Server Ready API </a>
+* <a href="#kfs-server-metadata">Server Metadata API </a>
+* <a href="#kfs-model-ready">Model Ready API </a>
+* <a href="#kfs-model-metadata">Model Metadata API </a>
+* <a href="#kfs-model-infer"> Inference API </a>
+
+## Server Live API <a name="kfs-server-live"></a>
+**Description**
+
+Get information about server liveness.
+
+**URL**
+
+```Bash
+GET http://${REST_URL}:${REST_PORT}/v2/health/live
```

Comment:
should this be executed or only curl commands? It's either missing "export REST_URL" commands or there shouldn't be Bash keyword here

Reply:
Removed Bash keyword. I think commands in this file should not be executed in test environment.